src/main/java/az/edu/turing/entities/BookingEntity.java

```diff
@@ -0,0 +1,48 @@
+package az.edu.turing.entities;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+public class BookingEntity implements Serializable {
+    private String bookingId;
+    private FlightEntity flight;
+    private List<PassengerEntity> passengers;
+
```

Comment:
also you can  add the status field so that it receives the values ​​from the Status enum